test/JsonApiDotNetCore.MongoDb.Example.Tests/IntegrationTests/Filtering/FilterOperatorTests.cs

```diff
@@ -0,0 +1,409 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web;
+using FluentAssertions;
+using Humanizer;
+using JsonApiDotNetCore.Configuration;
+using JsonApiDotNetCore.Queries.Expressions;
+using JsonApiDotNetCore.Serialization.Objects;
+using Microsoft.Extensions.DependencyInjection;
+using MongoDB.Driver;
+using Xunit;
+
+namespace JsonApiDotNetCore.MongoDb.Example.Tests.IntegrationTests.Filtering
+{
+    public sealed class FilterOperatorTests : IClassFixture<IntegrationTestContext<TestableStartup>>
+    {
+        private readonly IntegrationTestContext<TestableStartup> _testContext;
+
+        public FilterOperatorTests(IntegrationTestContext<TestableStartup> testContext)
+        {
+            _testContext = testContext;
+            
+            _testContext.RegisterResources(builder =>
+            {
+                builder.Add<FilterableResource, string>();
+            });
+            
+            _testContext.ConfigureServicesAfterStartup(services =>
+            {
+                services.AddResourceRepository<MongoDbRepository<FilterableResource>>();
+            });
+
+            var options = (JsonApiOptions) _testContext.Factory.Services.GetRequiredService<IJsonApiOptions>();
+            options.EnableLegacyFilterNotation = false;
+            options.SerializerSettings.DateFormatString = "yyyy-MM-dd";
```

Comment:
Dunno if this is the only solution but it's the only way I could work around tests failing because of the timezone I'm at.

```
JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.Filtering.FilterOperatorTests.Can_filter_comparison_on_DateTime.Can_filter_comparison_on_DateTime(matchingDateTime: "2001-01-01", nonMatchingDateTime: "2001-01-09", filterOperator: LessOrEqual, filterDateTime: "2001-01-05")

Xunit.Sdk.XunitException
Expected responseDocument.ManyData[0].Attributes["someDateTime"] to be "2001-01-01", but found <2001-01-01 03:00:00>.
   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.ObjectAssertions.Be(Object expected, String because, Object[] becauseArgs)
   at JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.Filtering.FilterOperatorTests.Can_filter_comparison_on_DateTime(String matchingDateTime, String nonMatchingDateTime, ComparisonOperator filterOperator, String filterDateTime) in /Users/mrnkr/Documents/code/JsonApiDotNetCore.MongoDb/test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/Filtering/FilterOperatorTests.cs:line 226
   at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_1.<<InvokeTestMethodAsync>b__1>d.MoveNext() in C:\Dev\xunit\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 264
--- End of stack trace from previous location where exception was thrown ---
   at Xunit.Sdk.ExecutionTimer.AggregateAsync(Func`1 asyncAction) in C:\Dev\xunit\xunit\src\xunit.execution\Sdk\Frameworks\ExecutionTimer.cs:line 48
   at Xunit.Sdk.ExceptionAggregator.RunAsync(Func`1 code) in C:\Dev\xunit\xunit\src\xunit.core\Sdk\ExceptionAggregator.cs:line 90
```